content/components.md

```diff
@@ -79,7 +79,7 @@ export class CatsController {
   }
 
   @Get()
-  async findAll(): Promise<Cat[]> {
+  async findAll(): Cat[] {
```

Comment:
it's an async function, a `Promise` will be returned back. I don't think it makes sense to remove it. In fact, an `await` should be added.